packages/verkle/src/node/internalNode.ts

```diff
@@ -29,11 +29,17 @@ export class InternalNode extends BaseVerkleNode<VerkleNodeType.Internal> {
     // Not implemented yet
   }
 
-  setChild(index: number, child: VerkleNode) {
+  // Updates the commitment value for a child node at the corresponding index
+  setChild(index: number, child: Uint8Array) {
     this.children[index] = child
+    // TODO: Update commitment as well
```

Comment:
Isn't `child` the commitment that is assigned to `children[index]` ? At least that's what I understand from line 13-14:

```
  // Array of commitments to child nodes
  public children: Uint8Array[]
```

Reply:
Yes indeed.  Thanks for catching that.  Will adjust.